### PR TITLE
Modify getting moment photos.

### PR DESCRIPTION
**Pull request by tyokujin**
In IOS13, [PHAssetCollectionType.moment ](https://developer.apple.com/documentation/photokit/phassetcollectiontype/moment) is deprecated.
`NohanaImagePicker.shouldShowMoment` is fetching photos from `PHAssetCollectionType.moment`.
https://github.com/nohana/NohanaImagePicker/blob/e14cc3670537275aa82a0ce44f8cb80bd4eb9805/NohanaImagePicker/PhotoKitAlbumList.swift#L67
I wrote fetching photos process that don't use `PHAssetCollectionType.moment`, for support iOS14~.

With this change, location information of header has been deleted.

### CODE REVIEW

File: NohanaImagePicker/MomentDetailListViewController.swift [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        }
+    }
+    
+    func didChangeAssetDetailPage(_ indexPath: IndexPath) {
+        guard let nohanaImagePickerController = nohanaImagePickerController, let momentInfoSection = momentInfoSection else {
+            return
+        }
+        let asset = PhotoKitAsset(asset: momentInfoSection.assetResult[indexPath.item])
+        pickButton.isSelected = nohanaImagePickerController.pickedAssetList.isPicked(asset)
+        pickButton.isHidden = !(nohanaImagePickerController.canPickAsset(asset) )
```
Comment (a2444mo10) [inline NohanaImagePicker/MomentDetailListViewController.swift:96]:
```suggestion
        pickButton.isHidden = !(nohanaImagePickerController.canPickAsset(asset))
```
Remove space.

File: NohanaImagePicker/MomentDetailListViewController.swift [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+        nohanaImagePickerController.delegate?.nohanaImagePicker?(nohanaImagePickerController, assetDetailListViewController: self, didChangeAssetDetailPage: indexPath, photoKitAsset: asset.originalAsset)
+    }
+
+    func scrollCollectionView(to indexPath: IndexPath) {
+        let count: Int? = momentInfoSection?.assetResult.count
+        guard count != nil && count! > 0 else {
+            return
+        }
```
Comment (a2444mo10) [inline NohanaImagePicker/MomentDetailListViewController.swift:104]:
```suggestion
        guard count = momentInfoSection?.assetResult.count, count > 0 else {
            return
        }
```

File: NohanaImagePicker/MomentViewController.swift [hunk 3]
```diff
@@ -149,16 +187,17 @@ class MomentViewController: AssetListViewController, ActivityIndicatable {
...
+        momentDetailListViewController.currentIndexPath = selectedIndexPath
     }
 
     // MARK: - IBAction
 
-    @IBAction override func didPushDone(_ sender: AnyObject) {
-        super.didPushDone(sender)
+    @IBAction func didPushDone(_ sender: AnyObject) {
+        let pickedPhotoKitAssets = nohanaImagePickerController!.pickedAssetList.map { ($0 as! PhotoKitAsset).originalAsset }
+        nohanaImagePickerController!.delegate?.nohanaImagePicker(nohanaImagePickerController!, didFinishPickingPhotoKitAssets: pickedPhotoKitAssets )
```
Comment (a2444mo10) [inline NohanaImagePicker/MomentViewController.swift:200]:
```suggestion
        if let nohanaImagePickerController = nohanaImagePickerController {
            let pickedPhotoKitAssets = nohanaImagePickerController.pickedAssetList.map { ($0 as! PhotoKitAsset).originalAsset }
            nohanaImagePickerController.delegate?.nohanaImagePicker(nohanaImagePickerController, didFinishPickingPhotoKitAssets: pickedPhotoKitAssets)
        }
```

File: NohanaImagePicker/MomentDetailListViewController.swift [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+        let asset = PhotoKitAsset(asset: momentInfoSection.assetResult[currentIndexPath.item])
+        if pickButton.isSelected {
+            if nohanaImagePickerController!.pickedAssetList.drop(asset: asset) {
+                pickButton.isSelected = false
+            }
+        } else {
+            if nohanaImagePickerController!.pickedAssetList.pick(asset: asset) {
+                pickButton.isSelected = true
+            }
+        }
```
Comment (a2444mo10) [inline NohanaImagePicker/MomentDetailListViewController.swift:197]:
```suggestion
        guard let nohanaImagePickerController = nohanaImagePickerController,
              let momentInfoSection = momentInfoSection else { return }
        let asset = PhotoKitAsset(asset: momentInfoSection.assetResult[currentIndexPath.item])
        if pickButton.isSelected {
            if nohanaImagePickerController.pickedAssetList.drop(asset: asset) {
                pickButton.isSelected = false
            }
        } else {
            if nohanaImagePickerController.pickedAssetList.pick(asset: asset) {
                pickButton.isSelected = true
            }
        }
```
`Forced Unwrapping`が気になりました。

**Review by a2444mo10 — APPROVED** [review]
`Forced Unwrapping`を使用している部分などで`Suggested change`を数件しましたが、機能的には問題ないと思います。

**Comment by tyokujin** [comment]
Thank you for your reviews!
I fixed code.

**Pull request merged**